po/ja.po

```diff
@@ -1783,13 +1848,60 @@ msgid ""
 "   run `cargo` commands, it will automatically download and compile missing\n"
 "   dependencies for you."
 msgstr ""
+"1. 「Copy to clipboard」でコードをコピー。\n"
+"\n"
+"2. `cargo new exercise`で`exercise/`ディレクトリを作成：\n"
+"\n"
+"    ```shell\n"
+"    $ cargo new exercise\n"
+"         Created binary (application) `exercise` package\n"
+"    ```\n"
+"\n"
+"3. `exercise/`ディレクトリに移動し、`cargo run`でバイナリをビルドして実行：\n"
+"\n"
+"    ```shell\n"
+"    $ cd exercise\n"
+"    $ cargo run\n"
+"       Compiling exercise v0.1.0 (/home/mgeisler/tmp/exercise)\n"
+"        Finished dev [unoptimized + debuginfo] target(s) in 0.75s\n"
+"         Running `target/debug/exercise`\n"
+"    Hello, world!\n"
+"    ```\n"
+"\n"
+"4. `src/main.rs`のボイラープレートコードを、コピーしたコードで置き換えてください。例えば、前のペー"
+"ジの例を使った場合、`src/main.rs`は以下のようになります。\n"
+"\n"
+"    ```rust\n"
+"    fn main() {\n"
+"        println!(“Edit me!”);\n"
```

Comment:
Here, `“` and `”` are used instead of `"`.  (U+201C, U+201D v.s. U+0022)
This caused a build  error on the CI.

Reply:
done

pls note: I edited directly on ja.po as poedit was automatically rewriting;
\"Edit me!\" to “Edit me!”

Reply:
@mgeisler Just FYI, it seems that poedit automatically rewrote double quotes. Did you encounter this issue?

Reply:
> @mgeisler Just FYI, it seems that poedit automatically rewrote double quotes. Did you encounter this issue?

Not me personally, but people reported this in the past. There is a setting somewhere that let's you turn it off.

Reply:
morning, just sharing in case it may help others with similar issues;

- auto-conversion is a Mac setting
- to disable, follow these steps;

1. Open poedit
2. Go to Edit > Substitutions
3. Uncheck "Smart Quotes"
Note: the focus may need to be in the "Translation" section to see toggle enabled
<img width="584" alt="Screenshot 2023-06-11 at 10 52 25" src="https://github.com/google/comprehensive-rust/assets/132030351/bd8bba3e-4620-460e-9769-237dcb7411e9">

Reply:
Thanks for that! I've quoted this in #800 and will refer others to that post in the future!